From 24e256ab005c68a3894845ef3b7764338f9442a4 Mon Sep 17 00:00:00 2001
From: Nikolaus Waxweiler <madigens@gmail.com>
Date: Fri, 4 Aug 2017 08:25:31 +0200
Subject: [truetype] Fix metrics of B/W hinting in v40 mode.

Phantom points are now saved outside v40 backwards compatibility
mode.  This fixes the jumping glyphs when switching between v35 and
v40 monochrome mode.

* src/truetype/ttgload.c (TT_Hint_Glyph): Fix inversed bool logic.

diff --git a/src/truetype/ttgload.c b/src/truetype/ttgload.c
index 086ebca52..bfda76175 100644
--- a/src/truetype/ttgload.c
+++ b/src/truetype/ttgload.c
@@ -825,7 +825,7 @@
     /* compatibility mode, where no movement on the x axis means no reason */
     /* to change bearings or advance widths.                               */
     if ( !( driver->interpreter_version == TT_INTERPRETER_VERSION_40 &&
-            !loader->exec->backward_compatibility ) )
+            loader->exec->backward_compatibility ) )
     {
 #endif
       loader->pp1 = zone->cur[zone->n_points - 4];
